core/common/junit/build.gradle.kts

```diff
@@ -31,6 +31,8 @@ dependencies {
 
     implementation(libs.junit.pioneer)
     implementation(libs.testcontainers.junit)
+
+    testFixturesImplementation(libs.mockito.core)
```

Comment:
Hi, why is this added?

Reply:
This was for moving MockMonitorExtension from src/test to  src/testFixtures in order to use it from added test in [the first commit](https://github.com/eclipse-edc/Connector/pull/3463/commits/a6a0dcccb216eabee434c7b58d750c707f6564f3). It can be reverted if we add no test here.

Reply:
Sorry, I'm a bit confused. That dependency no longer appears to be required based on the state of the current PR. Am I missing something? Where is the added test in the current PR state (not previous commits that appear to be partially reverted)?

Reply:
This change was removed by https://github.com/eclipse-edc/Connector/pull/3463/commits/1cb54eebb92bf6076691c15deac40064c49a83ab.